lib/Doctrine/Import/Builder.php

```diff
@@ -179,6 +179,14 @@ class Doctrine_Import_Builder extends Doctrine_Builder
      */
     protected $_phpDocEmail = '##EMAIL##';
 
+
+    /**
+     * Contains the actAs columns after running buildSetUp
+     *
+     * @var array
+     */
+    protected $_actAsColumns = array();
```

Comment:
You can make it private, as I do not see any current usage on child class.

Reply:
done